src/cmd/run.js

```diff
@@ -2,6 +2,7 @@
 import type FirefoxProfile from 'firefox-profile';
 import type Watchpack from 'watchpack';
 
+import {desktopNotifications as notifier} from '../util/desktop-notifier';
```

Comment:
how about aliasing this as `defaultDesktopNotifications` instead of `notifier`? (like we do for `defaultFirefoxConnector` at line 7)